railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_2.rb.tt

```diff
@@ -5,3 +5,6 @@
 # Once upgraded flip defaults one by one to migrate to the new default.
 #
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# button_to view helpers consistently render <button> elements
+# Rails.application.config.action_view.button_to_generates_button_tag = false
```

Comment:
@kaspth I've pushed up a commit to support the configuration.

> This'll also read better in `new_framework_defaults_6_2.rb`

Is this the line change you had in mind?

Reply:
This should be true so old apps generated before 6.2 can keep their old `input` default. But then they can uncomment to opt-in to the new default.

```suggestion
# Rails.application.config.action_view.button_to_generates_button_tag = true
```

We need to tweak `load_defaults` to set this to `true` in 6.2, automatically. That way `load_defaults 6.1` (and lower) in `config/application.rb` will get input tags, but can opt-in, while `load_defaults 6.2` get `button` tags.

Reply:
```ruby
# button_to view helpers consistently render <button> elements
```

Let's singularize "view helpers" since this only affects `button_to`, no? Let's strike consistently, reads as if the config is unreliable somehow.